Sources/DeploymentTargetIoTRuntime/IoTRuntime.swift

```diff
@@ -47,7 +47,9 @@ public class IoTRuntime<Service: WebService>: DeploymentProviderRuntime {
     }
     
     public func configure(_ app: Application) throws {
-        app.http.address = .hostname(currentNodeCustomLaunchInfo.host.path, port: currentNodeCustomLaunchInfo.port)
+        app.storage[HTTPConfigurationStorageKey.self] = HTTPConfiguration(
+            bindAddress: .interface(currentNodeCustomLaunchInfo.host.path,port: currentNodeCustomLaunchInfo.port)
+                 )
```

Comment:
@PSchmiedmayer Can you verify if this is the correct migration from the old `http` property? I'm not too familiar with the recent changes and don't want to introduce any bugs.

Reply:
The initializer looks good. Instead of assigning it to the storage key I would suggest to call the `configure` method which might do more than just assign it to the storage key.